daprdocs/content/en/operations/resiliency/policies.md

```diff
@@ -2,7 +2,7 @@
 type: docs
 title: "Policies"
```

Comment:
Should we called this "Resiliency policies"? Thoughts?

Reply:
I'm good with that - I like just "Policies" in the TOC though, so I'll leave that as is and just update the docs title